### PR TITLE
explorer, rmdir, winget: add traditional Chinese translation

### PR DESCRIPTION
**Pull request by poychang**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh_TW/windows/winget.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+`winget install {{package}}`
+
+- 顯示指定封裝的相關資訊：
+
+`winget show {{package}}`
+
+- 搜尋指定封裝：
+
+`winget search {{package}}`
```
Comment (kbdharun) [inline pages.zh_TW/windows/winget.md:16]:
```suggestion
`winget search {{套件}}`
```

File: pages.zh_TW/windows/winget.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> Windows 封裝管理員的命令列工具。
+> 更多資訊：<https://learn.microsoft.com/windows/package-manager/winget>.
+
+- 安裝一個封裝：
+
+`winget install {{package}}`
+
+- 顯示指定封裝的相關資訊：
+
+`winget show {{package}}`
```
Comment (kbdharun) [inline pages.zh_TW/windows/winget.md:12]:
```suggestion
`winget show {{套件}}`
```

File: pages.zh_TW/windows/winget.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# winget
+
+> Windows 封裝管理員的命令列工具。
+> 更多資訊：<https://learn.microsoft.com/windows/package-manager/winget>.
+
+- 安裝一個封裝：
+
+`winget install {{package}}`
```
Comment (kbdharun) [inline pages.zh_TW/windows/winget.md:8]:
```suggestion
`winget install {{套件}}`
```

File: pages.zh_TW/windows/rmdir.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`rmdir {{目錄/完整/路徑}}`
+
+- 遞迴刪除一個目錄和裡面的內容：
+
+`rmdir {{目錄/完整/路徑}} /s`
+
+- 在沒有提示的情況下遞迴刪除目錄和裡面的內容：
+
+`rmdir {{path/to/directory}} /s /q`
```
Comment (kbdharun) [inline pages.zh_TW/windows/rmdir.md:16]:
```suggestion
`rmdir {{目錄/完整/路徑}} /s /q`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page. 

[Reference](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**